api/v1alpha1/releaseplan_types.go

```diff
@@ -17,60 +17,55 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"github.com/redhat-appstudio/release-service/kcp"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// ReleaseLinkSpec defines the desired state of ReleaseLink.
-type ReleaseLinkSpec struct {
-	// DisplayName refers to the name of the ReleaseLink to link a user and managed workspace together
-	// +required
+// ReleasePlanSpec defines the desired state of ReleasePlan.
+type ReleasePlanSpec struct {
+	// DisplayName is the long name of the ReleasePlan
+	// +optional
 	DisplayName string `json:"displayName"`
 
 	// Application is a reference to the application to be released in the managed workspace
 	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
 	// +required
 	Application string `json:"application"`
 
-	// Target is a reference to the workspace to establish a link with
-	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
+	// Target references where in the cluster to find the information required for a release
 	// +required
-	Target string `json:"target"`
-
-	// Release Strategy defines which strategy will be used to release the application in the managed workspace. This field has no effect for ReleaseLink resources in unmanaged workspaces
-	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
-	// +optional
-	ReleaseStrategy string `json:"releaseStrategy,omitempty"`
+	Target kcp.NamespaceReference `json:"target"`
 }
 
-// ReleaseLinkStatus defines the observed state of ReleaseLink.
-type ReleaseLinkStatus struct {
+// ReleasePlanStatus defines the observed state of ReleasePlan.
+type ReleasePlanStatus struct {
 }
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="Display Name",type=string,priority=1,JSONPath=`.spec.displayName`
 // +kubebuilder:printcolumn:name="Application",type=string,JSONPath=`.spec.application`
-// +kubebuilder:printcolumn:name="Target",type=string,JSONPath=`.spec.target`
-// +kubebuilder:printcolumn:name="Release Strategy",type=string,JSONPath=`.spec.releaseStrategy`
+// +kubebuilder:printcolumn:name="NamespaceReference Namespace",type=string,priority=1,JSONPath=`.spec.target.namespace`
+// +kubebuilder:printcolumn:name="NamespaceReference Workspace",type=string,priority=1,JSONPath=`.spec.target.workspace`
```

Comment:
the name being Target Workspace makes way more sense to me

Reply:
Agree.